html-forms/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>
+      HTML Forms
+    </title>
+  </head>
+  <body>
+    <h1>
+      Coding Form!
+    </h1>
+    <form action="">
+      <div>
+        <label for="email">Email</label>
+        <div>
+          <input type="email" name="email" />
+        </div>
+      </div>
+      <div>
+        <label for="reasons">Why is coding cool?</label>
```

Comment:
As counter-intuitive as it is, a `<label>`'s `for` attribute refers to a form control by `id`, not by `name`: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label